extensions/amp-carousel/0.2/amp-carousel.css

```diff
@@ -127,3 +129,10 @@ amp-carousel {
 .i-amphtml-carousel-scroll[horizontal="false"] > .i-amphtml-carousel-spacer{
   width: 100%;
 }
+
+.i-amphtml-carousel-slotted > .i-amphtml-replaced-content {
+  /*
+   * Apply contain object-fit to all replaced content to avoid distorted ratios.
+   */
+  object-fit: contain;
```

Comment:
Do we need to design this to be overridable? Is there any situation in which we wouldn't want this to be default behavior?

Reply:
Yep, this should be overridable. `object-fit: cover` feels more natural to me, but this matches the behavior of carousel v1. A related issue is #20208, since this selector is a bit confusing for people to override.